pkg/config/config.go

```diff
@@ -118,4 +118,9 @@ type Config struct {
 
 	// used for development purposes only
 	RunSyncLocal string
+
+	// enriched values which are not present in the external API representation
+	TenantID       string
```

Comment:
@kargakis thinking we should stash these approximately here - the alternatives I saw were: in the public-facing API (not great), or carting around a different serialisation of the ContainerService (ugh).  wdyt?

Reply:
`node-startup.sh` is still using `.ContainerService.Properties.AzProfile`. Do you want to change that to use these?

Reply:
I did wonder about doing that; will do.